promscale/installation/upgrade.md

```diff
@@ -0,0 +1,70 @@
+# Upgrade Promscale
+Promscale consists of the Promscale Connector, and the Promscale PostgreSQL
+extension. When you upgrade your Promscale installation, you need to check both
+components. This section contains important information about upgrading.
+
+The process for updating your Promscale installation is:
+
+1.  Check that the most recent required version of the Promscale extension is
+    available.
+2.  Stop all Promscale Connector instances connected to the database.
+3.  Perform the upgrade.
+4.  Start the new version of the Promscale Connector, and wait for it to begin
+    ingesting data.
+5.  Start all other Promscale Connector instances.
+
+## Upgrade to Promscale&nbsp;0.11.0
+Promscale&nbsp;0.11.0 contains significant changes, and the upgrade drops any
+previous tracing data you have stored. Make a backup of your installation, and
+test the new version, before you go ahead with this upgrade.
+
+Upgrading to Promscale&nbsp;0.11.0 could take up to 5-10 minutes on very large
+databases. During the upgrade, Promscale is not able to ingest or query data.
+
+The upgrade process modifies a lot of database objects. On large deployments,
+you might need to increase the PostgreSQL `max_locks_per_transaction` parameter
+before you start the upgrade.
+
+<highlight type="warning">
+When you upgrade to Promscale&nbsp;0.11.0, all previous tracing data is
```

Comment:
Do we purposely duplicate the content here with that from three paragraphs above?

Reply:
Yes. People have a nasty tendency to not read admonitions, so in the case of really important ones, I like to duplicate the content.